app/src/main/java/com/simplemobiletools/calculator/activities/AboutActivity.kt

```diff
@@ -0,0 +1,109 @@
+package com.simplemobiletools.calculator.activities
+
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.content.res.Resources
+import android.net.Uri
+import android.os.Bundle
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import android.view.View
+import android.widget.TextView
+import butterknife.BindView
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.simplemobiletools.calculator.BuildConfig
+import com.simplemobiletools.calculator.Config
+import com.simplemobiletools.calculator.R
+import java.util.*
+
+class AboutActivity : SimpleActivity() {
```

Comment:
remove whole AboutActivity and LicenseActivity, reuse them from the Commons library. You launch the About activity together with giving it the appropriate license IDs, so I know which licences should be displayed at LicenseActivity. Here is an example of what I mean https://github.com/SimpleMobileTools/Simple-Gallery/blob/master/app/src/main/kotlin/com/simplemobiletools/gallery/extensions/activity.kt#L69 . Dont forget declaring the 3 new activities in the manifest, like at https://github.com/SimpleMobileTools/Simple-Gallery/blob/master/app/src/main/AndroidManifest.xml#L76 . Also remove the no longer needed strings from all files.

Reply:
I added some missing licences in Commons 2.35.4

Reply:
What about a license for `com.github.yukuku:ambilwarna:2.0.1`?

Reply:
Found ColorPickerDialog in Commons, will remove this dependency completely.

Reply:
ye ambilwarna is removed completely, I rewrote the color picker dialog so hex code can be inserted directly